Trimble.Modus.Components/Controls/Button/TMButton.cs

```diff
@@ -0,0 +1,407 @@
+        }
+
+        public ImageSource IconSource
+        {
+            get => (ImageSource)GetValue(IconSourceProperty);
+            set => SetValue(IconSourceProperty, value);
+        }
+
+        public Command Command
+        {
+            get => (Command)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        public object CommandParameter
+        {
+            get => GetValue(CommandParameterProperty);
+            set => SetValue(CommandParameterProperty, value);
+        }
+
+        public Color TextColor
+        {
+            get { return (Color)GetValue(TextColorProperty); }
+            set { SetValue(TextColorProperty, value); }
+        }
+
+        public Color BorderColor
+        {
+            get { return (Color)GetValue(BorderColorProperty); }
+            set { SetValue(BorderColorProperty, value); }
+        }
+
+        public bool IsDisabled
+        {
+            get { return (bool)GetValue(IsDisabledProperty); }
+            set { SetValue(IsDisabledProperty, value); }
+        }
+
+
+        public new Color BackgroundColor
+        {
+            get { return (Color)GetValue(BackgroundColorProperty); }
+            set { SetValue(BackgroundColorProperty, value); }
+        }
+
+
+        public int Radius
+        {
+            get { return (int)GetValue(ButtonRadiusProperty); }
+            set { SetValue(ButtonRadiusProperty, value); }
+        }
+
+
+        public TMButton()
+        {
+
+            _titleLabel = new Label
+            {
+                TextColor = Colors.White,
+            };
+
+            _iconImage = new Image
+            {
+                WidthRequest = 24,
+                HeightRequest = 24,
+
+            };
+            HorizontalOptions = LayoutOptions.Start;
+            setDefault(this);
+
+            stackLayout = new StackLayout();
+            _iconImage.SetBinding(Image.SourceProperty, new Binding(nameof(IconSource), source: this));
+            _titleLabel.VerticalOptions = LayoutOptions.Center;
+            stackLayout.Orientation = StackOrientation.Horizontal;
+            _titleLabel.SetBinding(Label.TextProperty, new Binding(nameof(Title), source: this));
+            stackLayout.Children.Add(_iconImage);
+            stackLayout.Children.Add(_titleLabel);
+            frame = new Border
+            {
+                Padding = 0,
+
+                Content = stackLayout,
+                BackgroundColor = (Color)BaseComponent.colorsDictionary()["TrimbleBlue"],
+                HorizontalOptions = LayoutOptions.Start,
+                StrokeShape = new Rectangle
+                {
+                    RadiusX = 4,
+                    RadiusY = 4
+                }
+            };
+
+
+
+            Content = frame;
+            
+
+            GestureRecognizers.Add(_tapGestureRecognizer = new TapGestureRecognizer());
+            _tapGestureRecognizer.Tapped += OnTapped;
```

Comment:
@ElishaSamPeterPrabhu  We should unsubscribe in the Dispose method.

Reply:
OnDispose Method not found some other way must be implemented to unsubscribe